src/model/quotes.js

```diff
@@ -164,13 +164,18 @@ class QuotesModel {
       throw ErrorHandler.CreateInternalServerFSPIOPError('Missing quoteRequest', null, fspiopSource)
     }
 
-    await this.db.getParticipant(fspiopSource, LOCAL_ENUM.PAYER_DFSP, quoteRequest.amount.currency, ENUM.Accounts.LedgerAccountType.POSITION)
-    await this.db.getParticipant(fspiopDestination, LOCAL_ENUM.PAYEE_DFSP, quoteRequest.amount.currency, ENUM.Accounts.LedgerAccountType.POSITION)
-
     // Following is the validation to make sure valid fsp's are used in the payload for simple routing mode
     if (envConfig.simpleRoutingMode) {
       await this.db.getParticipant(quoteRequest.payer.partyIdInfo.fspId, LOCAL_ENUM.PAYER_DFSP, quoteRequest.amount.currency, ENUM.Accounts.LedgerAccountType.POSITION)
```

Comment:
I think this is where the original bug was. We need to check that `quoteRequest.payer.partyIdInfo.fspId` and `quoteRequest.payee.partyIdInfo.fspId` exist before talking to the database.

The trouble was, that this was undefined, and passing that undefined through to knex.

Reply:
Thanks @lewisdaly, that was left there by accident.

I was doing some internal testing using the unit tests. It is now removed.